services/provider/server/server.go

```diff
@@ -423,6 +420,48 @@ func (s *OCSProviderServer) getExternalResources(ctx context.Context, consumerRe
 
 	}
 
+	cbpList := &rookCephv1.CephBlockPoolList{}
+	err = s.client.List(ctx, cbpList, client.InNamespace(s.namespace))
+	if err != nil {
+		return nil, fmt.Errorf("failed to list cephBlockPools in namespace. %v", err)
+	}
+	blockPoolMapping := []csiopv1a1.BlockPoolIdPair{}
+	for i := range cbpList.Items {
+		cephBlockPool := cbpList.Items[i]
+		if cephBlockPool.GetAnnotations()[util.BlockPoolMirroringTargetIDAnnotation] != "" {
+			blockPoolMapping = append(
+				blockPoolMapping,
+				[]string{
+					strconv.Itoa(cephBlockPool.Status.PoolID),
+					cephBlockPool.GetAnnotations()[util.BlockPoolMirroringTargetIDAnnotation],
+				},
+			)
+		}
+	}
+
+	if len(blockPoolMapping) != 0 {
```

Comment:
Length cannot be smaller then 0
```suggestion
	if len(blockPoolMapping) > 0 {
```